keyboards/kc60se/config.h

```diff
@@ -83,7 +82,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
  * power-up.
  *
  */
-//#define FORCE_NKRO
+#define FORCE_NKRO
```

Comment:
I'm not a fan of this being forced on, since it can prevent you from accessing the early boot stuff.

Reply:
I guess I can live with it off by default.